gemfirexd/core/src/drda/java/com/pivotal/gemfirexd/internal/impl/drda/ClientThread.java

```diff
@@ -104,6 +104,7 @@ public Object run() throws IOException
                         else {
                           clientSocket.setKeepAlive(false);
                         }
+                        clientSocket.setTcpNoDelay(true);
```

Comment:
I think for large results, if data is written continuously on the socket, more packets will be formed and sent but shouldn't affect performance.